tests/fixtures/ember-container-query-typescript/output/ember-container-query/tsconfig.json

```diff
@@ -1,6 +1,10 @@
 {
   "extends": "@tsconfig/ember/tsconfig.json",
   "compilerOptions": {
+    "declaration": true,
+    "declarationDir": "declarations",
+    "emitDeclarationOnly": true,
+    "noEmit": false,
```

Comment:
`glint --declaration` sets the compiler options `declaration`, `emitDeclarationOnly`, and `noEmit` (to `true`, `true`, and `false`, respectively) for the end-developer.

https://github.com/typed-ember/glint/blob/1.0.2/packages/core/src/cli/options.ts#L13-L16

For TS-only projects (i.e. without Glint), it is up to the end-developer to set these compiler options.